climada/entity/exposures/litpop/nightlight.py

```diff
@@ -439,15 +439,12 @@ def read_bm_file(bm_path, filename):
         Additional info from which coordinates can be calculated.
     """
     path = Path(bm_path, filename)
-    try:
-        LOGGER.debug('Importing %s.', path)
-        curr_file = gdal.Open(str(path))
-        band1 = curr_file.GetRasterBand(1)
-        arr1 = band1.ReadAsArray()
-        del band1
```

Comment:
@NicolasColombi : do you know why `band1` was deleted in the previous version? Why is the deletion skipped now?
(`del something` summons the garbage collector)

Reply:
Hi @emanuel-schmid, Yes, it is because in the old version extracting the first band of the image (`band1`) and then reading it as an array was done in 3 steps: 1) get the raster band , convert raster to array, delate the raster `band1` in non array format. By getting the first band and directly reading it as an array (line 446) nothing is stored, so no need to delate anything. Line 446 of the new version does the same as Lines 445 to 447 of the old.

Reply:
🙈 Thanks!